db/migrate/20220812205204_add_publication_date_to_work_version.rb

```diff
@@ -0,0 +1,7 @@
+class AddPublicationDateToWorkVersion < ActiveRecord::Migration[7.0]
+  def change
+    change_table :work_versions do |t|
+      t.datetime :published_at
```

Comment:
Do we want to pre-populate for existing work versions?

Reply:
I don't think so. We never will re-publish them so I don't see a reason to do so.